src/CMakeLists.txt

```diff
@@ -196,6 +196,12 @@ target_sources(viamcpp
       ${PROTO_GEN_DIR}/gen/robot/v1/robot.pb.h
 )
 
+set_target_properties(
```

Comment:
This gets us a soname containing the ABI version and an on-disk runtime file containing the full version, then a dev symlink.